### ISSUE TITLE
Acct. No. In Descriptive Record

### ISSUE THREAD

**Issue by jph** [zeptofs/aba#9]
Hello. Not sure if this is still looked at/maintained, just had a quick q.

I'm migrating from an old PHP ABA generator. We've been sending ABA files with an account number next to the BSB in the descriptive record. I've noticed this gem doesn't support that. Any ideas on that?

Thanks.

**Comment by kiela** [comment]
Hello @jph 

I'm not sure what you mean by "descriptive record". If you want to create a transaction, you first need to create a batch. Once batch is created, you can add a transaction to that batch and while doing so, you can specify BSB and Account Number:

```ruby
aba = Aba.batch financial_institution: 'ANZ', user_name: 'Joe Blow', user_id: 123456, process_at: 200615
aba.add_transaction bsb: '123-456', account_number: '000-123-456', amount: 50000
```

Does that answer your question?

**Comment by jph** [comment]
Hello.

https://github.com/andrba/aba/blob/22ea35e6ffae604230faa556a3d2e0986214bbea/lib/aba/batch.rb#L112

Notice in this section how there's quite a bit of padding. The old PHP library actually put an account number in this padding.

I've tested your library as you've indicated and NAB accepted the ABA perfectly. I think this may have been an artifact of the old PHP library I used to use.

I appreciate your response nonetheless, thank you.
